build.gradle

```diff
@@ -5,14 +5,41 @@ buildscript {
     }
 }
 
+plugins {
+    id 'com.jfrog.bintray' version '1.2'
+    id 'net.researchgate.release' version '2.1.2'
+}
+
 allprojects {
     apply plugin: 'idea'
+    apply plugin: 'java'
+    apply plugin: 'scala'
+    apply plugin: 'maven'
+    apply plugin: 'maven-publish'
+    apply plugin: 'com.jfrog.bintray'
 }
 
-subprojects {
+release {
```

Comment:
For the record (wasn't obvious for me): the README for `net.researchgate.release` says it'll only work in a multi-project build if the plugin is only applied to the root project, so we need to do this here instead of inside `subprojects`.